### DISCUSSION TITLE
Does this support NUCLEO-WL55JC1 as well?

### DISCUSSION THREAD

**Discussion by harshahegde17** [Lora-net/LoRaMac-node#1607] [General]
Hi I'm new to STM32, please let me know whether this repository supports NUCLEO-WL55JC1 which has stm32wl55jc

**Comment by danak6jq** [comment]
I do not believe there is support in this repo (LoRaMac-node) for the STM32WLxx parts, however, ST has a firmware package available for the STM32WLxx parts in CubeMX/CubeIDE that include a snapshot of LoRaMac-node with additional functionality

**Comment by harshahegde17** [comment]
Thank You
